py.typed

```diff
@@ -0,0 +1,2 @@
+# This file adds type hints to the wheel published to pypi and enables other
+# projects importing exasol.bucketfs without ignoring typing.
```

Comment:
Instead, add this information to the commit message. Generally, the purpose of this file should be well known, otherwise it should be looked up.

```suggestion
```